src/_playground/Routes.js

```diff
@@ -270,59 +273,117 @@ const routes = [
+                return (
+                    <ul className='frDocs-Nav__list' key={section.name}>
+                        <li className='frDocs-Nav__headers'>{section.name}</li>
+                        {sectionRoutes.map(route => (
+                            <li key={route.name}>
+                                <NavLink
+                                    activeClassName='frDocs-Nav__item--active'
+                                    className='frDocs-Nav__item'
+                                    key={route.url}
+                                    to={{ pathname: route.url }}>
+                                    {route.name}
+                                </NavLink>
+                            </li>
+                        ))}
+                    </ul>
+                );
+            }
+        });
+
+        const noItemsFound = (
+            <ul className='frDocs-Nav__list'>
+                <li className='frDocs-Nav__item'>
+                    No results found.
+                </li>
             </ul>
         );
-    });
 
-    return (
-        <BrowserRouter basename={process.env.PUBLIC_URL}>
-            <div className='frDocs-Container'>
-                <div className='frDocs-Sidebar'>
-                    <h1 className='frDocs-Logo'>FUNDAMENTAL REACT</h1>
-                    <nav className='frDocs-Nav'>
-                        {navItems}
-                    </nav>
-                </div>
-                <div className='frDocs-Content'>
-                    <Switch>
-                        {routes.map(route => {
-                            return (
-                                <Route
-                                    component={route.component}
-                                    exact
-                                    key={route.url}
-                                    path={route.url} />
-                            );
-                        })}
-                        <Redirect exact from=''
-                            to='/home' />
-                    </Switch>
+        return (
+            <BrowserRouter basename={process.env.PUBLIC_URL}>
+                <div className='frDocs-Container'>
+                    <div className='frDocs-Sidebar'>
+                        <h1 className='frDocs-Logo'>FUNDAMENTAL REACT</h1>
+                        <nav className='frDocs-Nav'>
+                            {freezeItems}
+                            <div className='frDocs-Nav__inputGroup'>
+                                <InputGroup
+                                    inputId='frDocs-Nav__search'
+                                    inputPlaceholder='Search'
+                                    inputType='search'
+                                    inputValue={this.state.query}
+                                    onChange={this.onChangeHandler} />
+                            </div>
+                            {this.state.filteredItems.length === 1 ? noItemsFound : navItems}
```

Comment:
Should this be hardcoded to 1 or should it take into account how many items are `omitSearch: true`?

Reply:
okay, will do this.

Reply:
I just double-checked and added more omitSearch:true items and it still works 🤷‍♀️. I'm going to play around with this a little more and see how this is actually happening.

Reply:
Okay, I believe this may be resolved now. :)